README.md

```diff
@@ -69,22 +69,37 @@ source .venv/bin/activate
 pip install -r requirements-dev.txt
 ```
 
+## Instructions
+
+You can set up and deploy the sample application on LocalStack by executing the commands in our Makefile. First, create a `.env` file using the provided `.env.example` file as a template, and include your LocalStack token in it. Then, run `make -s start` to initiate LocalStack on your machine.
+
+Next, execute `make -s terraform-setup` to provision the infrastructure on LocalStack using Terraform CLI and its scripts. Alternatively, run `make -s awslocal-setup` to set up the infrastructure with the local AWS CLI.
```

Comment:
As a result, yes, it won't leak the token because `-s` will prevent `make` from printing the commands as they are executed.
 I typically write `Makefiles` for local use, I think in this case the command can be run both: with and without `-s`.

Reply:
I could have just added `@` to the command.. I have changed this part in the `Makefile`